docs/source/reservations/reservations.rst

```diff
@@ -170,32 +170,18 @@ In case you want to specify the DWS configuration for each job/cluster, you can
 
 .. _dws-on-gke:
 
-Using DWS on GKE with Kueue
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Using DWS on GKE
+~~~~~~~~~~~~~~~~~
 
-DWS is also supported on Google Kubernetes Engine (GKE) with Kueue. To enable DWS on GKE, you need to set up your GKE cluster with Kueue and DWS; see the `GCP official docs <https://cloud.google.com/kubernetes-engine/docs/how-to/provisioningrequest>`__.
+GKE supports the two kinds of flex-start configurations:
 
-To launch a SkyPilot cluster or job on GKE with DWS, you can specify the DWS configuration in the SkyPilot task YAML:
```

Comment:
Remove the example usage here so we do not need to maintain duplicate contents in different files.

Reply:
Better to keep it to make doc self-contained. People don't like jumping around docs.

Reply:
Updated